sources/functions/rutorrent

```diff
@@ -146,6 +147,53 @@ function rutorrent_install() {
 RUC
 }
 
+function rutorrent_fm_config() {
```

Comment:
Would it be simpler to just `sed` the required changes into the file after the installation? I'd rather make changes post-install so that we won't have to keep updating this hardcoded config in mind.

Reply:
The configuration we are editing would look terrible with `sed`. The user would not be able to understand how to edit.
```
    $config['archive']['type'] = [
        '7z' => [
            'bin' => '7zip',
            'compression' => [1, 5, 9],
        ]];
```

Reply:
Ah, when I say "after installation", I mean within the scope of the script, and not something that a user needs to do manually. So you'd let the "standard" config install itself first, and once you know that file is present, you just sed it within the install script.

Reply:
I'm saying the user may want to edit this file. We should keep the formatting intact for them. `sed` is not designed to replace things like this. It works better for one liners not arrays.

Reply:
Yeah they can edit it as much as they can post-install, nothing wrong with that, we can still make changes to it either way.

We could also just append that config at the end of the file, there's ways

Reply:
Yeah they can edit it as much as they can post-install, nothing wrong with that, we can still make changes to it either way.

We could also just append that config at the end of the file, there's ways

Reply:
Okay, I will echo text to the end of the file next week. It's a config variable that can be reassigned.

Reply:
@flying-sausages @liaralabs We can't use `sed` or `echo` here. It does not work. It's required to use the `cat` method like all the other files. If this is too messy in bash, we can put it into the `patches` folder and replace the file. 

Please advise about next steps. I just wasted an hour trying to fix something that wasn't broken.

Reply:
I'm still a bit hesitant to merge this as-is. `cat`ing entire config files isn't my favourite approach to keeping things easy to maintain long term.

I'm trying to understand the difference between this config and the one shipped for amd64. Is it basically the removal of these lines?

```
$pathToExternals['rar'] = '';
and
    'rar' => [
        'bin' => 'rar',
        'compression' => [0, 3, 5],
    ]
```

Reply:
Yes, that is correct. We are removing lines for the unsupported `rar` binary.